implementation/src/main/java/io/smallrye/mutiny/operators/multi/MultiRetryOp.java

```diff
@@ -22,7 +21,7 @@
 
     public MultiRetryOp(Multi<? extends T> upstream, long times) {
         super(upstream);
-        this.times = ParameterValidation.positive(times, "times");
```

Comment:
Is this validation happening somewhere else now?

Isn't "safer" to keep it, or was it causing issues?

Reply:
It's happening in the caller class. I just moved it.

Reply:
Do we need to be concerned with other callers calling this, without that check in place?